python/ray/_private/usage/usage_lib.py

```diff
@@ -244,6 +244,8 @@ def _put_library_usage(library_usage: str):
 class TagKey(Enum):
     _TEST1 = auto()
     _TEST2 = auto()
+    _RLLIB_FRAMEWORK = auto()
```

Comment:
You don't have to start with `_` :)  `_TEST1` is meant for private testing so it starts with `_`.

Reply:
oh, copy & paste is the only way I know :)
changed.